WcaOnRails/app/models/regulation.rb

```diff
@@ -0,0 +1,16 @@
+class Regulation < SimpleDelegator
+  begin
+    REGULATIONS = JSON.parse(File.read("app/views/search_results/wca-regulations.json"))
```

Comment:
In that case I have two questions.

1. Is silently swallowing the exception really the best we can do?

2. How do we handle updates of this file? With this code, It looks like the server needs to be restarted each time.